src/app/modules/user/components/project-page/project-page.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-project-page',
+  templateUrl: './project-page.component.html',
+  styleUrls: ['./project-page.component.scss']
+})
+
+export class ProjectPageComponent implements OnInit {
+  titleProject: string = 'Меркурий'
+  descriptionProject: string = 'Депортамент цифровых решений'
+  detailsProject: Array<string> = ['Роскосмос', 'В работе', '26 октября 2019', '233 дня', '19 человек']
```

Comment:
I recommend using an object type with key-value data. For example: {customer: "Роскосмос", status: "В работе", projectStart: "26 октября 2019", duration: "233 дня", team: "19 человек"}; It's better that way because other developers can't figure out what this array of elements means.